OJP/OJP_FareSupport.xsd

```diff
@@ -139,45 +139,342 @@
+				<xs:annotation>
+					<xs:documentation>Product can only be booked by contacting specific authorised retail outlets</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="other">
+				<xs:annotation>
+					<xs:documentation>Other booking process.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="BookingProcessListOfEnumerations">
+		<xs:annotation>
+			<xs:documentation>List of Booking Process values.</xs:documentation>
+		</xs:annotation>
+		<xs:list itemType="BookingProcessEnumeration"/>
+	</xs:simpleType>
+	<!-- =================================================== -->
+	<xs:simpleType name="GroupBookingEnumeration">
+		<xs:annotation>
+			<xs:documentation>Allowed values for Group Booking.</xs:documentation>
+		</xs:annotation>
+		<xs:restriction base="xs:normalizedString">
+			<xs:enumeration value="groupsAllowed"/>
+			<xs:enumeration value="groupsNotAllowed"/>
+			<xs:enumeration value="groupsAllowedWithReservation"/>
+			<xs:enumeration value="groupBookingsRestricted"/>
+			<xs:enumeration value="unknown"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="GroupBookingListOfEnumerations">
+		<xs:annotation>
+			<xs:documentation>List of Group Booking values.</xs:documentation>
+		</xs:annotation>
+		<xs:list itemType="GroupBookingEnumeration"/>
+	</xs:simpleType>
+	<xs:complexType name="BookingArrangementsStructure" abstract="false">
+		<xs:annotation>
+			<xs:documentation>Type for BOOKING ARRANGEMENTs modified from NeTEx.</xs:documentation>
+		</xs:annotation>
+		<xs:group ref="BookingArrangementsGroup"/>
+	</xs:complexType>
+	<xs:complexType name="MultilingualString">
+		<xs:annotation>
+			<xs:documentation>Type for a string in a specified language.</xs:documentation>
+		</xs:annotation>
+		<xs:simpleContent>
+			<xs:extension base="xs:normalizedString">
+				<xs:attribute name="lang" type="xs:language">
+					<xs:annotation>
+						<xs:documentation>Language of string contents.</xs:documentation>
+					</xs:annotation>
+				</xs:attribute>
+				<xs:attribute name="textIdType" type="xs:normalizedString">
+					<xs:annotation>
+						<xs:documentation>Resource id of string.</xs:documentation>
+					</xs:annotation>
+				</xs:attribute>
+			</xs:extension>
+		</xs:simpleContent>
+	</xs:complexType>
+	<xs:group name="BookingArrangementsGroup">
+		<xs:annotation>
+			<xs:documentation>Elements for BOOKING ARRANGEMENTs.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="BookingContact" type="ContactDetailsStructure" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Contact for Booking. +v1.1</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="BookingMethods" type="BookingMethodListOfEnumerations" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Allowed Ways of Making a BOOKING.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
```

Comment:
The bookingAccess was skipped ... however it may be usefull to say that booking is not open to everybody (only for disabled, or for specific card holders for example)